metadata.json

```diff
@@ -118,7 +116,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.7.0 < 8.0.0"
+      "version_requirement": ">= 6.0.0 < 8.0.0"
```

Comment:
We should still let people upgrade their older nodes using this module. So I think we should not update this.